api/argo-templates/ncn/storage.rebuild.yaml

```diff
@@ -0,0 +1,108 @@
+#
```

Comment:
note that this folder is only used by test. our real workflow should live in docs-csm repo

Reply:
Noted